src/cypress/integration/Form/Checkbox/checkbox-disabled.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { checkboxCopy } from 'shared/data/copyContent';
+
+describe('checkbox-disabled-test', () => {
```

Comment:
Let's update this to the way you have it in the `checkbox-default` tests. It should read: "Checkbox Disabled tests".